CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.3.0] - September, 14, 2022
+
+### React Components
+
+#### Added
+
+- [CookieConsent] component for React
```

Comment:
Should we separate the CookieConsent components here? There would be then 2 components: CookiePage and CookieModal?